content/security.md

```diff
@@ -103,16 +126,15 @@ You will get this answer.
 +----------------------------------------------------+----------------------------------------------------+-------------------+
 | Client id                                          | Secret                                             |  Label            |
 +====================================================+====================================================+===================+
-| 3e2iqilq2ygwk0ccgogkcwco8oosckkkk4gkoc0k4s8s044wss | 44ectenmudus8g88w4wkws84044ckw0k4w4kg0sokoss84oko8 |                   |
-| 4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s | 5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s |                   |
-| 4_5e6kfzmath8gowk0s000kkgc0o44cwgwsockwk0ccss4sw0w | 2nwha9mzk2w4so0cgokwocswoc48s0sg44wgg40kkokgg4w0go | Magento_connector |
+| 3e2iqilq2ygwk0ccgogkcwco8oosckkkk4gkoc0k4s8s044wss | 44ectenmudus8g88w4wkws84044ckw0k4w4kg0sokoss84oko8 | ERP connector |
+| 4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s | 5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s | Print catalog connector |
+| 4_5e6kfzmath8gowk0s000kkgc0o44cwgwsockwk0ccss4sw0w | 2nwha9mzk2w4so0cgokwocswoc48s0sg44wgg40kkokgg4w0go | Magento connector |
 +----------------------------------------------------+----------------------------------------------------+-------------------+
 ```
-As you can see, if you created your pair of client id / secret with a label, it will appear here.
 
 ### Get a token
 
-A client id has been created and provided to the client application. The last information needed to use the API is the token.
+An API connection was created and its client Id and secret were provided to the client application. The last information needed to use the API is the token.
```

Comment:
client id (if you want to be constistent with all the doc)

Reply:
:+1: